content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md

```diff
@@ -0,0 +1,90 @@
+
+### The new set of Data Services APIs
+
+This set of APIs family are broken down into different groups:
+
+1. **Data Services :** Group of APIs to accommodate common resources used for both Data Services and Infrastructure Platform suites such as Task List (Asynchronous Operations), Dual Authorization, Issues, Settings, Storage Locations, and displaying DSCC Tags
+2. **Virtualization :** Group of APIs to accommodate common interaction with both on-premises and public cloud Hyper-Visors such as registration of VMware vCenter or AWS account, discovery of virtual machines or AWS EC2 instance, migration of virtual machines and many others use cases.
+3. **Backup and Recovery :** Group of APIs designed to deploy and register the Data Orchestrator, the Protection Store Gateway, to create protection policy for VM, DataStores, Volumes, Microsoft SQL Server, AWS EC2, AWS EBS, and many other use cases in HPE GreenLake for Backup and Recovery.
+4. **Private Cloud Business Edition :** Group of APIs designed to view inventory of a DHCI 2.0, SimpliVity, or Azure cloud account, to perform upgrades for Storage software, hyper-Visor software, to add, update delete provisioning policies and many other use cases in HPE GreenLake for Private Cloud Business Edition.
+
+### The new documentation about the family of Data Services APIs
+
+This is the pivoting point for the HPE GreenLake APIs for Data Services on the HPE GreenLake platform where HPE GreenLake user’s clients for API can perform more complex automation that involves all categories of services made available in the Data Services on HPE GreenLake platform. To accommodate the documentation for the combination of multiple sets of APIs, the GreenLake documentation about the APIs had been updated to provide directions on how to access the family of the HPE GreenLake API for Data Services on HPE GreenLake platform shown in this [link](https://support.hpe.com/hpesc/public/docDisplay?docId=sd00003533en_us&page=ps_api_dscc.html). The help menu inside the HPE GreenLake for those data services as of March 2024 was also updated as shown below.
```

Comment:
Second sentence, add "HPE" in front of GreenLake documentation about the APIs...